staging/nvidia/charts/grafana/values.yaml

```diff
@@ -1,7 +1,2 @@
-#
-# grafana dashboards
-#
-enabled: true
```

Comment:
are users still able to disable this chart in cluster.yaml?

Reply:
yes, if you look at the updated values file ou will see that the grafana yaml stanza has the enabled field. You can set this to false and it will not deploy this chart. You may test it out by running helm install/template